osf/management/commands/create_test_notifications.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management.base import BaseCommand
+from osf.models.notifications import NotificationSubscription
+from osf.models import OSFUser, Node
+from django.utils.crypto import get_random_string
+from django.utils import timezone
+
+class Command(BaseCommand):
```

Comment:
This command is unnecessary; it doesn't actually create duplicates that collide on `_id`